code_samples/tutorials/page_tutorial/assets/css/style.css

```
@@ -235,7 +235,7 @@ footer div {
     margin-top: 15px;
 }
 
-.block-form .ezstring-field {
+.block-form .ibexa-string-field {
```

```suggestion
.block-form .ibexa_string-field {
```
See: https://github.com/ibexa/page-builder/blame/main/src/lib/Behat/Page/LandingPagePreview.php#L46